ngx_http_apicast_module.c

```diff
@@ -114,6 +116,32 @@ int ngx_http_apicast_ffi_set_proxy_cert_key(
 
   ctx->proxy_client_cert = x509;
   ctx->proxy_client_cert_key = cert_key;
+
+  if ( number_of_certs > 1 ) {
+    STACK_OF(x509) *chain = sk_X509_new_null();
```

Comment:
Note that the type above is `STACK_OF(X509)` and this is `STACK_OF(x509)`. Not familiar with this particular macro, but the difference might cause issues.